lib/suites/LinkedDataSignature.js

```diff
@@ -89,6 +89,19 @@ module.exports = class LinkedDataSignature {
     if(options.nonce !== undefined && typeof options.nonce !== 'string') {
       throw new TypeError('"options.nonce" must be a string.');
     }
+    const {signer} = options;
+    if(signer) {
+      if(!(signer.sign && typeof signer.sign === 'function')) {
+        throw new TypeError('"options.signer.sign" must be a function.');
+      }
+      if(!(signer.creator && typeof signer.creator)) {
```

Comment:
I'm wondering if `creator` should be passed to the sign function instead? I guess someone crafting the `sign` function could do this on their own ... any thoughts here? Seems natural to me that the `sign` function would receive the ID of the key to be used and the data to be signed ... and it would return an object that includes at least the raw signature (we may want to allow for it to include other meta data in the future, I don't know).